docker/test/stress/run.sh

```diff
@@ -110,7 +110,8 @@ function stop()
     # We failed to stop the server with SIGTERM. Maybe it hang, let's collect stacktraces.
     kill -TERM "$(pidof gdb)" ||:
     sleep 5
-    gdb -batch -ex 'thread apply all backtrace' -p "$(cat /var/run/clickhouse-server/clickhouse-server.pid)" ||:
+    echo "thread apply all backtrace (on stop)" >> /test_output/gdb.log
+    gdb -batch -ex 'thread apply all backtrace' -p "$(cat /var/run/clickhouse-server/clickhouse-server.pid)" | ts '%Y-%m-%d %H:%M:%S' >> /test_output/gdb.log
```

Comment:
I'm not sure if it's necessary, but shouldn't we disable signal handling to avoid gdb hanging, like it done here: https://github.com/ClickHouse/ClickHouse/blob/7378468b4a5d9487393c0dde5425af507299628d/docker/test/stress/run.sh#L145 )

Reply:
No, we should not, because:
- We disable signal handling for different reason: to ignore signals that can be used on normal server operation. Not to avoid gdb hanging.
- Disabling handling of some signals (like SIGTRAP) is actually the thing that may cause gdb to hang (and that's what's [the comment](https://github.com/ClickHouse/ClickHouse/blob/b238ed409385d8534d73f2ef9062c4a4f7609ffe/docker/test/stress/run.sh#L139-L140) about)